presto-product-tests/bin/run_on_docker.sh

```diff
@@ -35,6 +39,18 @@ function stop_unnecessary_hadoop_services() {
   docker exec ${HADOOP_MASTER_CONTAINER} supervisorctl stop zookeeper
 }
 
+function start_mapr_httpfs_service() {
+  HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
+  docker exec ${HADOOP_MASTER_CONTAINER} chown -R mapr /opt/mapr/httpfs
```

Comment:
make /opt/mapr as an env variable for MAPR_HOME  or something and use it everywhere else too.